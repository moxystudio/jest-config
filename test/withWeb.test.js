'use strict';

const configWeb = require('../addons/withWeb');

it('should match snapshot', () => {
    expect(configWeb()).toMatchSnapshot();
});
