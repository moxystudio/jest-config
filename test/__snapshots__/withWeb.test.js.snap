// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "collectCoverageFrom": Array [
    "www/**/*.js",
  ],
  "coveragePathIgnorePatterns": Array [
    ".*/.data/.js",
  ],
  "setupFilesAfterEnv": Array [
    "@testing-library/jest-dom/extend-expect",
  ],
  "transform": Object {
    "/.data-url/.": "<PROJECT_ROOT>/addons/withWeb/data-url.js",
    "/.inline/.": "<PROJECT_ROOT>/addons/withWeb/inline.js",
  },
}
`;
